### ISSUE TITLE
1.16.4 fluid autocrafting duplication bug

### ISSUE THREAD

**Issue by Thorjackie** [refinedmods/refinedstorage#2772]
#### Issue description:
fluid autocrafting 'duplicates' fluids stored both internally and externally when autocrafting by raising the 'stored' value of the fluid till autocrafting completes.

#### What happens:
when autocrafting with fluids, as soon as autocrafting begins, the 'stored' value of the fluid gradually goes up till it reaches a maximum ( which seems to be related to the amount of fluid storage present in the system ) or the autocrafting is complete. When autocrafting is done, the 'stored' value of the fluid remains, the 'crafting' won't complete until canceled, and all fluid containers, both fluid storage drives and external tanks, are filled with the 'stored' fluid, thusly duplicating it. 

#### What you expected to happen:
When autocrafting with fluid, X number of mB are either stored or processed, used up, and autocrafting completes when processing does.

#### Steps to reproduce:
1. create recipe that used fluid autocrafting (ie 1 empty bucket, 1 B lava, in a tank, to produce 1 lava bucket)
2. craft with said recipe, from one bucket to many
3. receive infinite fluid
...
(Note this seems to happen only on bigger systems, I tried it in a fresh world with a minimal RS system, and I had no issues. This has happened to me on RS 1.9.8 on MC 1.16.3 as well, both times it only occurred when my RS system got larger.)

#### Version (make sure you are on the latest version before reporting):

- Minecraft: 1.16.4
- Forge: 35.1.13
- Refined Storage: 1.9.9

Does this issue occur on a server? [yes/no]
yes ( tested on lan )

#### If a (crash)log is relevant for this issue, link it here:
no logs nor crashes are reported during the issue, however I can produce a  video if need be.

Screenshots:
Autocrafting Setup:
![autocrafting_setup](https://user-images.githubusercontent.com/48392309/102732749-22095f80-4301-11eb-8b54-662d6ab4bb65.png)

Autocraft Recipe:
![autocraft_recipe](https://user-images.githubusercontent.com/48392309/102732762-2b92c780-4301-11eb-8367-70e71f45e5cf.png)

Tank Initial Amount:
![tank_initial_amount](https://user-images.githubusercontent.com/48392309/102732774-32b9d580-4301-11eb-96cc-67f118a9f4b1.png)

Autocraft Mid-craft:
![autocraft_mid_craft](https://user-images.githubusercontent.com/48392309/102732789-3cdbd400-4301-11eb-83dc-9db927278e20.png)

autocraft finished:
![autocraft_finished](https://user-images.githubusercontent.com/48392309/102732807-48c79600-4301-11eb-8835-935b4c44637d.png)

tank after autocrafting and autocraft is cancelled:
![tank_autocraft_finished](https://user-images.githubusercontent.com/48392309/102732871-672d9180-4301-11eb-9241-a89c48fe74ef.png)

**Comment by Darkere** [comment]
Noneed for video. I've seen this on once on stream already :/. 
I was completely unable to reproduce it though.

Can you please make a list of things that are interacting with fluids in your system?

**Comment by Thorjackie** [comment]
Of course. I've used two setups to test this out in my main world. The first setup had a blast chiller and a fluid plenisher from thermal as autocrafting, my fluid storage was a mekanism tank. Trying to eliminate extraneous variables, removed the crafter on the blast chiller, and i replaced my fluid plenisher with a mekanism tank. The only other fluid external storage is connected to a cooking for blockheads sink for water. I replaced my storage tank with a 64k fluid storage drive and still had the same issue. When i get back i can see if removing the sink or disconnecting the bulk of my RS network has any effect. I will note, in my test world where it seemed to work fine, i also used a mekanism tank for storage and there were no issues.
